### DISCUSSION TITLE
Raspberrymatic keine Funktion mit RF USB Stick

### DISCUSSION THREAD

**Discussion by Stefan03888** [OpenCCU/OpenCCU#2607] [🇩🇪 Fragen&Antworten]
### Describe the issue you are experiencing

Hallo, habe gestern mein Homematic IP Local umgestellt auf Raspberrymatic mit RF USB Stick leider aber ohne Erfolg, seit heute gehen immer mehr Geräte in die Kommunikationsstörung. Kann leider kein Gerät mehr steuern über HA, habe die Geräte nicht neu angelernt sonder nur den Host geändert bei HA Integration auf die von Rasberrymatic

### Describe the behavior you expected

CCU3 Backup aufgespielt

### Steps to reproduce the issue

1.keine Kommunikation mit Funkstick zu den Geräten
2.Fertiggerät und nichts selber gelötet
3.Hostnamen nur geändert statt alles neu angelernt
...

### What is the version this bug report is based on?

3.73.9.20231130

### Which base platform are you running?

intelnuc (intel NUC)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
RaspberryMatic CCU
Mounting /data as /usr/local (Home Assistant Add-On): OK
Starting watchdog...
Identifying host system: Intel Corporation NUC7i5BNB (oci), OK
Initializing RTC Clock: onboard, OK
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: OK: backup successfully restored
Running seedrng: OK
Initializing System: OK
Setup ca-certificates: OK
Starting logging: OK
Init onboard LEDs: init, OK
Starting irqbalance: OK
Starting iptables: OK
Starting network: eth0: link up, fixed, firewall, inet up, 172.30.33.2, OK
Identifying Homematic RF-Hardware: ....HmRF: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:00:14.0-1, HmIP: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:00:14.0-1, OK
Updating Homematic RF-Hardware: HMIP-RFUSB: 4.4.18, not necessary, OK
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting ha-proxy: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: OK
Starting hs485d: disabled
Starting multimacd: .......OK
Starting rfd: .OK
Starting HMIPServer: ........OK
Starting ReGaHss: .OK
Starting CloudMatic: OK
Starting NeoServer: OK
Starting Third-Party Addons: OK
Starting crond: OK
Setup onboard LEDs: booted, OK
Finished Boot: 3.73.9.20231130 (raspmatic_oci_amd64)
```

**Comment by Baxxy13** [comment]
Der HmIP-RFUSB steckt direkt am NUC?
Was sagt der Carrier Sense auf der Startseite von RaspberryMatic?
Bittel mal für ein paar Minuten beobachten.

**Reply by Stefan03888** [reply]
Steckt mit nem 2m USB 3.0 Kabel im NUC, carrier sense bleibt bei 0% dauerhaft

**Reply by Stefan03888** [reply]
Es kommen stündlich neue Geräte ohne Kommunikation dazu im systemlog bei raspberry

**Reply by Stefan03888** [reply]
Es muss ja irgendwie am RF Stick liegen bezüglich mangelnder Kommunikation? Oder muss ich die Integration HM IP LAN löschen?

**Reply by Baxxy13** [reply]
Was sagt der Duty-Cycle, ist der auch dauerhaft 0%?

Da es den HmIP-RFUSB nur als Bausatz gibt muss ihn ja jemand zusammengelötet haben.
Hat derjenige die Funktionalität überprüft?

**Reply by Stefan03888** [reply]
Duty Cycle ist bei 32%, wurde online gekauft und auf der Verpackung steht geprüfte Ware

**Reply by Baxxy13** [reply]
OK, das ist ja schon mal was.
Betreffen die Komm-Störungen alle Geräte oder nur HmIP?

**Reply by Stefan03888** [reply]
Betrifft auch die Homematic Standard Geräte, gibt es noch irgendeine Möglichkeit den Stick zu prüfen?

**Reply by Baxxy13** [reply]
So richtig prüfen kann man den Stick leider nicht.
Alle "Fehler-Theorien" die ich zum Thema hatte konnten durch deine Antworten weitestgehend ausgeschlossen werden.

Stell mal in der Systemsteuerung -> Zentralenwartung das Fehlerprotokoll für "Homematik Funk" (und bitte nur das!) auf "alles loggen".
Das lässt du mal 5Min so und wenn du kannst bedienst du währenddessen ein HM-Gerät direkt am Gerät (Schaltaktor/ Rolladen etc.).

Anschließend das Fehlerprotokoll (gleiche Stelle wie eben) herunterladen und hier anhängen. Vielleicht sieht man was.
Das Logging dann wieder zurückstellen auf "nur Fehler".

**Reply by blackbasket** [reply]
Hey ihr zwei... Ich hatte das gleiche Verhalten einfach mal nach einem Update... ohne HW-Änderung oder sonst was... da hat einfach ein restore geholfen... daher einfach mal der Vorschlag, dass initiale Backup einfach nochmal einzuspielen...

**Reply by Stefan03888** [reply]
Ok sende morgen mal das Protokoll rein. Und mit Initial Backup meinst du das von meiner CCU 3?

**Reply by blackbasket** [reply]
Jepp

**Reply by Stefan03888** [reply]
Ist auch noch nen versuch wert aber dann muss ich immer den RF USB Stick neu anlernen weil es den da ja nicht gibt

**Reply by Stefan03888** [reply]
So komischweise heute nochmal nachgeschaut und jetzt ist nur eine Gerätekommunikation gestört, alle anderen funktionieren über Raspberrymatic komischweise, habe die CCU 3 runtergefahren und den neuen Host bei der Homematic IP Local integration angegeben und jetzt geht es auch wieder, dabei hab ich nichts verändert

**Comment by Stefan03888** [comment]
[5422eb72-raspberrymatic-2024-01-13.log](https://github.com/jens-maus/RaspberryMatic/files/13927528/5422eb72-raspberrymatic-2024-01-13.log)

Habe aber trotzdem mal die Logging Datei reingepackt vielleicht sieht man ja noch was

**Reply by Baxxy13** [reply]
Man sieht nicht viel. 
Vom Homematic-Funk Debug gerade mal anderthalb Minuten in denen nichts relevantes zu sehen ist.
Vermutlich hat sich in der kurzen Zeit kein HM-Gerät gemeldet und du hast auch keins lokal "bedient".
Aber läuft ja anscheinend wieder. 😉  

Außerdem hast du was in deiner Firewall falsch eingestellt. Aber das nur am Rande.
```
Jan 12 08:48:15 5422eb72-raspberrymatic user.info firewall: iptables v1.8.9 (legacy): invalid port/service `//192.168.2.218:8123' specified Try `iptables -h' or 'iptables --help' for more information.
Jan 12 08:48:15 5422eb72-raspberrymatic user.info firewall: iptables v1.8.9 (legacy): Port "//192.168.2.218" does not resolve to anything.  Try `iptables -h' or 'iptables --help' for more information.
```

**Comment by Stefan03888** [comment]
Habe da aber ein homematic Gerät verwendet, vielleicht sollte ich es nochmal per schaltbefehl machen? 

Was ist in der Firewall falsch?

**Reply by Baxxy13** [reply]
> Was ist in der Firewall falsch?

Ja keine Ahnung wenn du keinen Screenshot zeigst. Ich habe nur auf die Fehlermeldung im Log hingewiesen.

**Reply by Stefan03888** [reply]
![Screenshot 2024-01-14 194329](https://github.com/jens-maus/RaspberryMatic/assets/152796908/251b16ab-2359-4c5e-9683-5fdf527e41ba)
![Screenshot 2024-01-14 194357](https://github.com/jens-maus/RaspberryMatic/assets/152796908/d8d5b607-033d-4203-933e-7bcc0d4889f1)

**Reply by blackbasket** [reply]
Also prinzipiell ist deine "Port-Freigabe" falsch... da gehört nur der Port rein... IP höchstens einen Dialog weiter unten... da du aber alle FW-Regeln deaktiviert hast, ist es ohnehin überflüssig ;)

**Reply by Baxxy13** [reply]
So ist es. 
:8123 ist der Homeassistant - UI Port, wüsste nicht warum man den in der RM-Firewall "freigeben" sollte.

**Reply by Stefan03888** [reply]
Also einfach rauslassen und gut?

**Reply by Baxxy13** [reply]
Ja, solange du alles auf "offen" und "Vollzugriff" hast braucht unten gar nichts rein.

**Reply by Stefan03888** [reply]
Ok erledigt danke

**Comment by Stefan03888** [comment]
[5422eb72-raspberrymatic-2024-01-13 (1).log](https://github.com/jens-maus/RaspberryMatic/files/13927795/5422eb72-raspberrymatic-2024-01-13.1.log)

hier nochmal der Log File über 5min

**Reply by Baxxy13** [reply]
Ist immer noch nix zu sehen, aber wenn's funktioniert ist ja alles gut.
Hier mal ein Beispiel von mir wo ich eine HM-Schaltsteckdose lokal schalte:
```
Jan 13 11:14:13 RM-Test-VM-96 user.debug rfd: RX for QEQ0011501: @38836641 RSSI=-35dB 0x6C95C8 -> 0xFFE3E9 INFO_ACTUATOR_STATUS [SEQ0896344]:   CNT=5,RPTEN=1,RPTED=0,BIDI=1,BURST=0,WAKEUP=0,WAKEMEUP=0,BCAST=1,TYPE=0x10   CHANNEL = 1   STATUS = 0   STATE = 0   CLOCK = 0   LOWBAT = 0   DUTY_CYCLE = 0   RSSI = 0
Jan 13 11:14:13 RM-Test-VM-96 user.debug rfd: Event: QEQ0011501:1.STATE=false
Jan 13 11:14:13 RM-Test-VM-96 user.debug rfd: Event: QEQ0011501:1.WORKING=false
```

**Reply by Stefan03888** [reply]
Heute ist es wieder passiert und ich habe das Problem gefunden nur noch keine langfristige Lösung und zwar stürzt irgendwie Raspberrymatic ab, denn nach einem neu starten des Addons ging dann wieder alles.
Ich hänge mal das Fehlerprotokoll mit an, vielleicht weiß einer ja was ich falsch eingestellt habe.

[home-assistant_2024-01-14T18-36-56.712Z.log](https://github.com/jens-maus/RaspberryMatic/files/13932701/home-assistant_2024-01-14T18-36-56.712Z.log)

**Reply by Baxxy13** [reply]
Man sieht nur das die Verbindung zum RaspberryMatic-AddOn weg ist. Was ja klar ist wenn das AddOn abstürzt.

Der Rest hat mit irgendwelchen Custom-Integrationen zu tun, und nem Fehler in der configuration.yaml.
Nichts was mit dem RM-AddOn zu tun hat.

**Reply by Stefan03888** [reply]
Kann man das noch irgendwie erforschen?^^

**Comment by heitec11** [comment]
Hallo,
ich hatte letzte Woche auch Probleme, dass immer mehr Geräte mit Gerätekommunikation nicht mehr erreichbar waren.
Hauptsachlich HmIP Geräte und daher meine zwei HAP gegeneinander getauscht, um zu sehen, ob der Fehler mitwandert.
Habe zufällig https://homeitems.de/ccu-geraetekommunikation-gestoert-fehlersuche-teil-1/ gefunden und den HomematicManager aus Teil 2 installiert. (Version 2.7.0; die neuere Version 2.7.1 läuft nicht bei mir und auch vielen anderen)

Letztendlich sieht es bei mir so aus, das Rasbperrymatic 3.73.9.20231130 die Ursache war.
Nach einem Downgrade auf 3.71.12.20231020 läuft es bei mir wieder Fehlerfrei.

Bei dem Stick ist es wichtig eine stabile Versorgungsspannung und Abstand mit ein USB Verlängerungskabel zu verwenden.

PS:Aktuell bereite ich meinen Stick auf folgenden Umbau vor: https://github.com/wolwin/WW-my3DP/tree/master/3DP_HmIP-RFUSB
